data/src/main/java/org/oppia/data/backends/gae/ModifyJsonInterceptor.kt

```diff
@@ -0,0 +1,30 @@
+package org.oppia.data.backends.gae
+
+import okhttp3.*
```

Comment:
Please expand imports instead of using wildcards. It helps avoid namespace pollution, and provides better context to readers on needed dependencies.

Reply:
Solved this